### ISSUE TITLE
Unable to build in Carthage using Xcode 8.3.1

### ISSUE THREAD

**Issue by craigotis** [ArtSabintsev/UIFloatLabelTextField#28]
I'm having issues building this framework using Carthage. My `Cartfile`:
```
github "Alamofire/Alamofire" ~> 4.0
github "SwiftyJSON/SwiftyJSON" ~> 3.0
github "hackiftekhar/IQKeyboardManager" ~> 4.0
github "JonasGessner/JGProgressHUD" ~> 1.0
github "ArtSabintsev/UIFloatLabelTextField" ~> 1.0
```
My build output:
```
craig@[~/projects/myapp/ios] carthage update --platform iOS
*** Fetching UIFloatLabelTextField
*** Fetching JGProgressHUD
*** Fetching IQKeyboardManager
*** Fetching SwiftyJSON
*** Fetching Alamofire
*** Checking out Alamofire at "4.4.0"
*** Checking out IQKeyboardManager at "v4.0.9"
*** Checking out JGProgressHUD at "v1.4.1"
*** Checking out SwiftyJSON at "3.1.4"
*** Checking out UIFloatLabelTextField at "1.2.7"
*** xcodebuild output can be found in /var/folders/hn/bglkhk1n71x9nlybv9ndqdsr0000gn/T/carthage-xcodebuild.LScCD9.log
*** Building scheme "Alamofire iOS" in Alamofire.xcworkspace
<unknown>:0: warning: argument unused during compilation: '-iapinotes-modules /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/lib/swift/iphoneos/arm64'
<unknown>:0: warning: argument unused during compilation: '-iapinotes-modules /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/lib/swift/iphoneos/arm64'
A shell task (/usr/bin/xcrun xcodebuild -project /Users/craig/projects/myapp/ios/Carthage/Checkouts/UIFloatLabelTextField/UIFloatLabelSampleApp/UIFloatLabelSampleApp.xcodeproj -scheme UIFloatLabelTextField -configuration Release CODE_SIGNING_REQUIRED=NO CODE_SIGN_IDENTITY= CARTHAGE=YES clean -showBuildSettings) failed with exit code 66:
xcodebuild: error: Scheme UIFloatLabelTextField is not currently configured for the clean action.
```
I've tried this with `--no-use-binaries`, just a regular `carthage update`, it always fails the same way. Removing this single framework, the build is fine again.

**Comment by ArtSabintsev** [comment]
Should be fixed now in version 1.2.8. There was a shared scheme that was broken/non-existant.

**Comment by craigotis** [comment]
With 1.2.8 I have a different issue:
```
*** Skipped building UIFloatLabelTextField due to the error:
Dependency "UIFloatLabelTextField" has no shared framework schemes
```

**Comment by ArtSabintsev** [comment]
gah, back tot he drawing board. I'll take a look this evening after work.

For the record, I'm a Cocoapods user.

**Comment by ArtSabintsev** [comment]
Should be fixed now in v1.2.9.
